decent-dao/fractal-interface
# Update ukrainian translations

## Description

This PR updates Ukrainian translations

## Notes

This PR rebased on #1023 to

## Issue / Notion doc (if applicable)

N/A

## Testing

1. Select Ukrainian language in the app
2. Click around
3. Make sure all the texts are translated
4. Slava Ukraine!

## Screenshots (if applicable)
